kpet/run.py

```diff
@@ -14,12 +14,14 @@
 """Module where the `run` command is implemented"""
 from __future__ import print_function
 from xml.sax.saxutils import escape, quoteattr
-from kpet.exceptions import ActionNotFound
+from kpet.exceptions import ActionNotFound, ParameterNotFound
 from kpet import utils
 
 
 def generate(args):
     """Generate an xml output compatible with beaker"""
+    if not args.db:
+        raise ParameterNotFound('--db is required')
```

Comment:
Isn't there a way to make the argument parser require the option for us?

Reply:
Ah, are you trying to keep it not required for other commands, such as `help`?

Reply:
exactly :/